newhelm/main.py

```diff
@@ -59,6 +59,29 @@ def list_tests() -> None:
         click.echo(f"Description: {metadata.description}")
         click.echo()
 
+@newhelm_cli.command()
+def list_suts() -> None:
+    """List details about all registered SUTs."""
+    secrets = load_secrets_from_config()
+    for sut, sut_entry in SUTS.items():
+        try:
+            sut_obj = sut_entry.make_instance(secrets=secrets)
+        except MissingSecretValues as e:
+            display_header(
+                f"Cannot display SUT {sut} because it requires the following secrets:"
+            )
+            for secret in e.descriptions:
+                click.echo(secret)
+            click.echo()
+            continue
+        
+        display_header(sut)
+        click.echo(f"class={sut_obj.__class__.__name__}")
+        click.echo(f"args={sut_obj._initialization_record.args}") # Consider create getter function for `_initialization_record`
```

Comment:
The `sut_entry` variable in this loop is a [FactoryEntry](https://github.com/mlcommons/newhelm/blob/bfba11f6383f9e7fdd0b114f4b896cc0be5df5e1/newhelm/instance_factory.py#L13C7-L13C19) which has all the args and kwargs in it already. You can get this information without attempting to construct the `sut_obj`.